sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -206,24 +210,28 @@ export interface KeyPollerOptions extends coreHttp.RequestOptionsBase {
 }
 
 // @public
-export interface KeyProperties extends ParsedKeyVaultEntityIdentifier {
+export interface KeyProperties {
     createdOn?: Date;
     enabled?: boolean;
     expiresOn?: Date;
-    id?: string;
+    name: string;
```

Comment:
name should be inaddition to ID, which includes vaultEndpoint, name, and version.

Reply:
Fixing it